assets/javascripts/openqa.js

```diff
@@ -58,9 +58,10 @@ function addUniqueFlash(status, id, text, container) {
     window.uniqueFlashMessages = {};
   }
   // update existing flash message
-  var existingFlashMessage = window.uniqueFlashMessages[id];
+  const existingFlashMessage = window.uniqueFlashMessages[id];
   if (existingFlashMessage) {
-    existingFlashMessage.find('span').first().text(text);
+    const element = typeof text === 'string' ? $('<span>' + text + '</span>') : text;
```

Comment:
Why is there a `typeof text === 'string'` check? I went through the calls of this function and didn't see a case where it was not a `string`. Also, why is the element created with `$()`?

Reply:
All is just in consistency with the normal `addFlash`.

Reply:
At least the `$(...)` doesn't do anything at all i think.

Reply:
I removed the `$(…)` in both functions.